examples/polling-parent/pom.xml

```diff
@@ -79,29 +79,17 @@
     <dependencies>
       <!-- API -->
       <dependency>
-        <groupId>org.apache.tomee</groupId>
-        <artifactId>javaee-api</artifactId>
-        <version>[8.0,)</version>
-        <scope>provided</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.tomee</groupId>
-        <artifactId>openejb-api</artifactId>
-        <version>${tomee.version}</version>
+        <groupId>org.apache.tomee.bom</groupId>
+        <artifactId>tomee-plus-api</artifactId>
+        <version>8.0.7-SNAPSHOT</version>
```

Comment:
Maybe use `tomee.version` instead of the hard string as it is already defined in the `properties` section. We also can remove `cxf.version` and `xbean.version` from the `<properties` section of the parent pom. 

In addition, we can remove

```
 <dependency>
        <groupId>org.apache.cxf</groupId>
        <artifactId>cxf-core</artifactId>
        <version>${cxf.version}</version>
      </dependency>
      <dependency>
        <groupId>org.apache.xbean</groupId>
        <artifactId>xbean-finder-shaded</artifactId>
        <version>${xbean.version}</version>
      </dependency>
      <dependency>
        <groupId>org.apache.xbean</groupId>
        <artifactId>xbean-reflect</artifactId>
        <version>${xbean.version}</version>
      </dependency>
```

as it comes with the related BOM.